addons/mail/tests/discuss/test_discuss_channel.py

```diff
@@ -199,6 +199,30 @@ def test_channel_info_seen(self):
             "Last message id should stay the same after mark channel as seen with an older message"
         )
 
+    @users('employee')
+    def test_set_last_seen_message_should_send_notification_only_if_channel_member_is_found(self):
```

Comment:
maybe a bit too long test bale, also "member is found" is too technical, just "once" is enough I think (more functional)